substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ReflectionConfigurationParser.java

```diff
@@ -282,19 +280,15 @@ private String formatMethod(T clazz, String methodName, List<T> paramTypes) {
         return delegate.getTypeName(clazz) + '.' + methodName + '(' + parameterTypeNames + ')';
     }
 
-    private void handleError(String message) {
+    private static void handleError(String message) {
         handleError(message, null);
     }
 
-    private void handleError(String msg, Throwable cause) {
+    private static void handleError(String msg, Throwable cause) {
         String message = msg;
         if (cause != null) {
             message += " Reason: " + formatError(cause) + '.';
         }
-        if (allowIncompleteClasspath) {
-            System.err.println("Warning: " + message);
-        } else {
-            throw new JSONParserException(message + " To allow unresolvable reflection configuration, use option --allow-incomplete-classpath");
-        }
+        System.err.println("Warning: " + message);
```

Comment:
Since `allowIncompleteClasspath` doesn't exist anymore (i.e. is now `true` per default)
```java
if (allowIncompleteClasspath) {
System.err.println("Warning: " + message);
} else {
    throw new JSONParserException(message + " To allow unresolvable reflection configuration, use option --allow-incomplete-classpath");
}
```
the else branch became dead code.